addons/metallb/00_metallb-namespace.yaml

```diff
@@ -0,0 +1,1883 @@
+# Copyright 2021 The Kubermatic Kubernetes Platform contributors.
```

Comment:
Please rename this file - something like just `00_metallb.yaml` should be fine. The other one could be `01_address-pools.yaml`.

Reply:
all right